compiler/noirc_evaluator/src/ssa/opt/inlining/inline_info.rs

```diff
@@ -365,3 +366,54 @@ fn mark_brillig_functions_to_retain(
         );
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::ssa::{ir::map::Id, ssa_gen::Ssa};
+
+    use super::compute_inline_infos;
+
+    // This panic is a regression that will need to be updated by fixing the inline infos computation.
+    // https://github.com/noir-lang/noir/issues/8448
+    #[test]
+    #[should_panic]
```

Comment:
Can you perhaps indicate the expected panic message, to give a hint as to what is wrong with the inline infos?

Reply:
Good idea, done